vita3k/gui/src/app_context_menu.cpp

```diff
@@ -154,60 +150,65 @@ void draw_app_context_menu(GuiState &gui, HostState &host) {
     // App Context Menu
     if (ImGui::BeginPopupContextItem("##app_context_menu")) {
         ImGui::SetWindowFontScale(1.3f);
-        if (ImGui::MenuItem("Boot", host.app_title.c_str()))
-            gui.app_selector.selected_title_id = host.io.title_id;
-        if (ImGui::MenuItem("Check App Compatibility")) {
-            const std::string compat_url = host.io.title_id.find("PCS") != std::string::npos ? "https://vita3k.org/compatibility?g=" + host.io.title_id : "https://github.com/Vita3K/homebrew-compatibility/issues?q=" + host.app_title;
-            system((OS_PREFIX + compat_url).c_str());
-        }
-        if (ImGui::BeginMenu("Copy App Info")) {
-            if (ImGui::MenuItem("ID and Name")) {
-                ImGui::LogToClipboard();
-                ImGui::LogText("%s [%s]", host.io.title_id.c_str(), host.app_title.c_str());
-                ImGui::LogFinish();
+        if (ImGui::MenuItem("Boot", host.app_short_title.c_str()))
+            run_app(gui, host);
+        if (host.io.title_id.find("NPXS") == std::string::npos) {
+            if (ImGui::MenuItem("Check App Compatibility")) {
+                const std::string compat_url = host.io.title_id.find("PCS") != std::string::npos ? "https://vita3k.org/compatibility?g=" + host.io.title_id : "https://github.com/Vita3K/homebrew-compatibility/issues?q=" + host.app_title;
+                system((OS_PREFIX + compat_url).c_str());
+            }
+            if (ImGui::BeginMenu("Copy App Info")) {
+                if (ImGui::MenuItem("ID and Name")) {
+                    ImGui::LogToClipboard();
+                    ImGui::LogText("%s [%s]", host.io.title_id.c_str(), host.app_title.c_str());
+                    ImGui::LogFinish();
```

Comment:
What about creating a function here?

Reply:
I have untouched code here so not sur understund your ask